static/bidder-info/zentotem.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "https://rtb.zentotem.net/bid?sspuid=cqlnvfk00bhs0b6rci6g"
+maintainer:
+  email: support@zentotem.net
```

Comment:
I sent email. Please response.

Reply:
@przemkaczmarek hello, we sent a response letter for verification